tests/selectedtests/test_app.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from selectedtests.app import app as under_test
+from flask import testing
+
+
+@pytest.fixture()
+def app_client():
+    """
+    Client for the flask web with mocked endpoints.
+    """
+    client = under_test.create_app().test_client()
```

Comment:
You mentioned not wanting to share app state across tests. You can avoid changing the app's architecture (`create_app` function wrapping everything) by changing this to:
```
@pytest.fixture()
def app_client():
    """
    Client for the flask web with mocked endpoints.
    """
    from selectedtests.app import app as under_test
    client = under_test.application.test_client()
    return client
```

Reply:
I don't want to have to re-import `selectedtests.app` module every time the `app_client` fixture is called.